### ISSUE TITLE
Navigating to tags via keyboard is not possible

### ISSUE THREAD

**Issue by ivanmcgregor** [select2/select2#3962]
For better accessibility it should be possible to target the tags in a multiselect; specifically the "x" to close it. Wrapping the x inside a link that is styled and has `tabindex="0"` could be a solution. 

A slight correction to my title: It is possible by first deleting all other tags with backspace, then remove the one tag originally destined to be removed and then add the others again... Not great.

**Comment by kevin-brown** [comment]
This is a duplicate of https://github.com/select2/select2/issues/910.
